packages/backend/src/routers/slackRouter.ts

```diff
@@ -0,0 +1,137 @@
+import {
+    ForbiddenError,
+    NotFoundError,
+    SlackSettings,
+} from '@lightdash/common';
+import { ExpressReceiver } from '@slack/bolt';
+import express from 'express';
+import * as fs from 'fs';
+import path from 'path';
+import { analytics } from '../analytics/client';
+import { LightdashAnalytics } from '../analytics/LightdashAnalytics';
+import { slackOptions } from '../clients/Slack/SlackOptions';
+import {
+    deleteInstallationFromOrganizationUuid,
+    getInstallationFromOrganizationUuid,
+} from '../clients/Slack/SlackStorage';
+import {
+    isAuthenticated,
+    unauthorisedInDemo,
+} from '../controllers/authentication';
+
+export const slackRouter = express.Router({ mergeParams: true });
+
+slackRouter.get(
+    '/',
+    isAuthenticated,
+    unauthorisedInDemo,
+
+    async (req, res, next) => {
+        try {
+            const organizationUuid = req.user?.organizationUuid;
+            if (!organizationUuid) throw new ForbiddenError();
+            const slackAuth = await getInstallationFromOrganizationUuid(
+                organizationUuid,
+            );
+            const response: SlackSettings = {
+                organizationUuid,
+                slackTeamName: slackAuth.installation?.team?.name || 'Slack',
+                createdAt: slackAuth.createdAt,
+            };
+            res.json({
+                status: 'ok',
+                results: response,
+            });
+        } catch (error) {
+            next(error);
+        }
+    },
+);
+
+slackRouter.get(
+    '/image/:imageId',
+
+    async (req, res, next) => {
+        if (!req.params.imageId.startsWith('slack-image')) {
+            throw new NotFoundError(`File not found ${req.params.imageId}`);
+        }
+        try {
+            const filePath = path.join('/tmp', req.params.imageId);
+            res.sendFile(filePath);
+        } catch (error) {
+            next(error);
+        }
+    },
+);
+
+slackRouter.delete(
+    '/',
+    isAuthenticated,
+    unauthorisedInDemo,
+
+    async (req, res, next) => {
+        try {
+            const organizationUuid = req.user?.organizationUuid;
+            if (!organizationUuid) throw new ForbiddenError();
+            await deleteInstallationFromOrganizationUuid(organizationUuid);
+
+            res.json({
+                status: 'ok',
+            });
+        } catch (error) {
+            next(error);
+        }
+    },
+);
+
+slackRouter.get(
+    '/install/:organizationUuid',
+    isAuthenticated,
+    unauthorisedInDemo,
+
+    async (req, res, next) => {
+        try {
+            const metadata = {
+                organizationUuid: req.params.organizationUuid,
+                userId: req.user?.userId,
+            };
```

Comment:
Should we use the `organizationUuid` from the user object? otherwise, they could create a connection to any org.